avm/res/cache/redis-enterprise/tests/e2e/amr/main.test.bicep

```diff
@@ -0,0 +1,49 @@
+targetScope = 'subscription'
+
+metadata name = 'Azure Managed Redis (Preview)'
+metadata description = 'This instance deploys an Azure Managed Redis (Preview) cache with the minimum set of required parameters.'
```

Comment:
As you call out "preview" - I recall that `redis-enterprise` was already implemented in the CARML days but we had to 'retire' it. We documented it as a ['known issue'[(https://github.com/Azure/ResourceModules/wiki/Known%20issues#cacheredis-enterprise) back then. @jtracey93 would you happen to know if we'd be able to deploy set resource in the AVM subscription? In the CARML one it was definitely not an option.

Reply:
Sidenote: Added it to our agenda for tomorrow